plugin.video.mlbtv/resources/lib/globals.py

```diff
@@ -0,0 +1,504 @@
+# coding=utf-8
+import sys, re, os, time
+import calendar, pytz
+import urllib, requests
+from datetime import date, datetime, timedelta
+from kodi_six import xbmc, xbmcplugin, xbmcgui, xbmcaddon
+
+if sys.version_info[0] > 2:
+    import http
+    cookielib = http.cookiejar
+    urllib = urllib.parse
+else:
+    import cookielib
+
+addon_handle = int(sys.argv[1])
+
+
+#Addon Info
+ADDON = xbmcaddon.Addon()
+ADDON_ID = ADDON.getAddonInfo('id')
+ADDON_VERSION = ADDON.getAddonInfo('version')
+ADDON_PATH_PROFILE = xbmc.translatePath(ADDON.getAddonInfo('profile'))
+KODI_VERSION = float(re.findall(r'\d{2}\.\d{1}', xbmc.getInfoLabel("System.BuildVersion"))[0])
+LOCAL_STRING = ADDON.getLocalizedString
+ROOTDIR = ADDON.getAddonInfo('path')
+
+#Settings
+settings = xbmcaddon.Addon(id='plugin.video.mlbtv')
+USERNAME = str(settings.getSetting(id="username"))
```

Comment:
Just for future reference, the preference is to use the newer settings calls (like getSettingString(id), getSettingBool(id), getSettingInt(id), and getSettingNumber(id)). More info at:

https://codedocs.xyz/xbmc/xbmc/group__python__xbmcaddon.html

Reply:
Got it, will add it now so it's ready for the next pr. Thanks for the review @pkscout!